privacyidea/static/components/config/views/config.policies.details.html

```diff
@@ -273,16 +273,6 @@ <h3 translate>Action</h3>
                 </label>
 
                 <div class="col-sm-9">
-                    <div isteven-multi-select
```

Comment:
Why did this have to go?

Reply:
As far as i could see, this isn't used and just left an empty drop-down something unless a "scope" was selected above. Then the accordion would show up.
I don't know what the intentions of this part were.
Tested only on Firefox...

Reply:
Right. This might be a remnant from when I restructured the UI of the policy details with the two tabs.